mkdir GetOrganelle-fungal-mitogenome-eval

# generate label table
slim_fastg.py -F fungus_mt `ls GetOrganelle-fungal-mitogenome/auto--SRR5803930-mitogenome/*.fasta | head -n1` --out-dir GetOrganelle-fungal-mitogenome-eval --out-base SRR5803930
slim_fastg.py -F fungus_mt `ls GetOrganelle-fungal-mitogenome/auto--SRR5804139-mitogenome/*.fasta | head -n1` --out-dir GetOrganelle-fungal-mitogenome-eval --out-base SRR5804139
slim_fastg.py -F fungus_mt `ls GetOrganelle-fungal-mitogenome/auto--SRR5802419-mitogenome/*.fasta | head -n1` --out-dir GetOrganelle-fungal-mitogenome-eval --out-base SRR5802419
slim_fastg.py -F fungus_mt `ls GetOrganelle-fungal-mitogenome/auto--SRR5804120-mitogenome/*.fasta | head -n1` --out-dir GetOrganelle-fungal-mitogenome-eval --out-base SRR5804120
slim_fastg.py -F fungus_mt `ls GetOrganelle-fungal-mitogenome/auto--SRR5804020-mitogenome/*.fasta | head -n1` --out-dir GetOrganelle-fungal-mitogenome-eval --out-base SRR5804020
slim_fastg.py -F fungus_mt `ls GetOrganelle-fungal-mitogenome/auto--SRR5803905-mitogenome/*.fasta | head -n1` --out-dir GetOrganelle-fungal-mitogenome-eval --out-base SRR5803905
slim_fastg.py -F fungus_mt `ls GetOrganelle-fungal-mitogenome/auto--SRR5764827-mitogenome/*.fasta | head -n1` --out-dir GetOrganelle-fungal-mitogenome-eval --out-base SRR5764827
slim_fastg.py -F fungus_mt `ls GetOrganelle-fungal-mitogenome/auto--SRR5804147-mitogenome/*.fasta | head -n1` --out-dir GetOrganelle-fungal-mitogenome-eval --out-base SRR5804147
slim_fastg.py -F fungus_mt `ls GetOrganelle-fungal-mitogenome/auto--SRR5804015-mitogenome/*.fasta | head -n1` --out-dir GetOrganelle-fungal-mitogenome-eval --out-base SRR5804015
slim_fastg.py -F fungus_mt `ls GetOrganelle-fungal-mitogenome/auto--SRR5804115-mitogenome/*.fasta | head -n1` --out-dir GetOrganelle-fungal-mitogenome-eval --out-base SRR5804115
slim_fastg.py -F fungus_mt `ls GetOrganelle-fungal-mitogenome/auto--SRR5754079-mitogenome/*.fasta | head -n1` --out-dir GetOrganelle-fungal-mitogenome-eval --out-base SRR5754079
slim_fastg.py -F fungus_mt `ls GetOrganelle-fungal-mitogenome/auto--SRR5803928-mitogenome/*.fasta | head -n1` --out-dir GetOrganelle-fungal-mitogenome-eval --out-base SRR5803928
slim_fastg.py -F fungus_mt `ls GetOrganelle-fungal-mitogenome/auto--SRR5765276-mitogenome/*.fasta | head -n1` --out-dir GetOrganelle-fungal-mitogenome-eval --out-base SRR5765276
slim_fastg.py -F fungus_mt `ls GetOrganelle-fungal-mitogenome/auto--SRR5804127-mitogenome/*.fasta | head -n1` --out-dir GetOrganelle-fungal-mitogenome-eval --out-base SRR5804127
slim_fastg.py -F fungus_mt `ls GetOrganelle-fungal-mitogenome/auto--SRR5801943-mitogenome/*.fasta | head -n1` --out-dir GetOrganelle-fungal-mitogenome-eval --out-base SRR5801943
slim_fastg.py -F fungus_mt `ls GetOrganelle-fungal-mitogenome/auto--SRR5802181-mitogenome/*.fasta | head -n1` --out-dir GetOrganelle-fungal-mitogenome-eval --out-base SRR5802181
slim_fastg.py -F fungus_mt `ls GetOrganelle-fungal-mitogenome/auto--SRR5803933-mitogenome/*.fasta | head -n1` --out-dir GetOrganelle-fungal-mitogenome-eval --out-base SRR5803933
slim_fastg.py -F fungus_mt `ls GetOrganelle-fungal-mitogenome/auto--SRR5801947-mitogenome/*.fasta | head -n1` --out-dir GetOrganelle-fungal-mitogenome-eval --out-base SRR5801947
slim_fastg.py -F fungus_mt `ls GetOrganelle-fungal-mitogenome/auto--SRR5804112-mitogenome/*.fasta | head -n1` --out-dir GetOrganelle-fungal-mitogenome-eval --out-base SRR5804112
slim_fastg.py -F fungus_mt `ls GetOrganelle-fungal-mitogenome/auto--SRR5804145-mitogenome/*.fasta | head -n1` --out-dir GetOrganelle-fungal-mitogenome-eval --out-base SRR5804145
slim_fastg.py -F fungus_mt `ls GetOrganelle-fungal-mitogenome/auto--SRR5802698-mitogenome/*.fasta | head -n1` --out-dir GetOrganelle-fungal-mitogenome-eval --out-base SRR5802698
slim_fastg.py -F fungus_mt `ls GetOrganelle-fungal-mitogenome/auto--SRR5804017-mitogenome/*.fasta | head -n1` --out-dir GetOrganelle-fungal-mitogenome-eval --out-base SRR5804017
slim_fastg.py -F fungus_mt `ls GetOrganelle-fungal-mitogenome/auto--SRR5804159-mitogenome/*.fasta | head -n1` --out-dir GetOrganelle-fungal-mitogenome-eval --out-base SRR5804159
slim_fastg.py -F fungus_mt `ls GetOrganelle-fungal-mitogenome/auto--SRR5804023-mitogenome/*.fasta | head -n1` --out-dir GetOrganelle-fungal-mitogenome-eval --out-base SRR5804023
slim_fastg.py -F fungus_mt `ls GetOrganelle-fungal-mitogenome/auto--SRR5803923-mitogenome/*.fasta | head -n1` --out-dir GetOrganelle-fungal-mitogenome-eval --out-base SRR5803923
slim_fastg.py -F fungus_mt `ls GetOrganelle-fungal-mitogenome/auto--SRR5803890-mitogenome/*.fasta | head -n1` --out-dir GetOrganelle-fungal-mitogenome-eval --out-base SRR5803890
slim_fastg.py -F fungus_mt `ls GetOrganelle-fungal-mitogenome/auto--SRR5764829-mitogenome/*.fasta | head -n1` --out-dir GetOrganelle-fungal-mitogenome-eval --out-base SRR5764829
slim_fastg.py -F fungus_mt `ls GetOrganelle-fungal-mitogenome/auto--SRR5803926-mitogenome/*.fasta | head -n1` --out-dir GetOrganelle-fungal-mitogenome-eval --out-base SRR5803926
slim_fastg.py -F fungus_mt `ls GetOrganelle-fungal-mitogenome/auto--SRR5764835-mitogenome/*.fasta | head -n1` --out-dir GetOrganelle-fungal-mitogenome-eval --out-base SRR5764835
slim_fastg.py -F fungus_mt `ls GetOrganelle-fungal-mitogenome/auto--SRR5802413-mitogenome/*.fasta | head -n1` --out-dir GetOrganelle-fungal-mitogenome-eval --out-base SRR5802413
slim_fastg.py -F fungus_mt `ls GetOrganelle-fungal-mitogenome/auto--SRR5801934-mitogenome/*.fasta | head -n1` --out-dir GetOrganelle-fungal-mitogenome-eval --out-base SRR5801934
slim_fastg.py -F fungus_mt `ls GetOrganelle-fungal-mitogenome/auto--SRR5803916-mitogenome/*.fasta | head -n1` --out-dir GetOrganelle-fungal-mitogenome-eval --out-base SRR5803916
slim_fastg.py -F fungus_mt `ls GetOrganelle-fungal-mitogenome/auto--SRR5803922-mitogenome/*.fasta | head -n1` --out-dir GetOrganelle-fungal-mitogenome-eval --out-base SRR5803922
slim_fastg.py -F fungus_mt `ls GetOrganelle-fungal-mitogenome/auto--SRR5804137-mitogenome/*.fasta | head -n1` --out-dir GetOrganelle-fungal-mitogenome-eval --out-base SRR5804137
slim_fastg.py -F fungus_mt `ls GetOrganelle-fungal-mitogenome/auto--SRR5801929-mitogenome/*.fasta | head -n1` --out-dir GetOrganelle-fungal-mitogenome-eval --out-base SRR5801929
slim_fastg.py -F fungus_mt `ls GetOrganelle-fungal-mitogenome/auto--SRR5803913-mitogenome/*.fasta | head -n1` --out-dir GetOrganelle-fungal-mitogenome-eval --out-base SRR5803913
slim_fastg.py -F fungus_mt `ls GetOrganelle-fungal-mitogenome/auto--SRR5801935-mitogenome/*.fasta | head -n1` --out-dir GetOrganelle-fungal-mitogenome-eval --out-base SRR5801935
slim_fastg.py -F fungus_mt `ls GetOrganelle-fungal-mitogenome/auto--SRR5803909-mitogenome/*.fasta | head -n1` --out-dir GetOrganelle-fungal-mitogenome-eval --out-base SRR5803909
slim_fastg.py -F fungus_mt `ls GetOrganelle-fungal-mitogenome/auto--SRR5801933-mitogenome/*.fasta | head -n1` --out-dir GetOrganelle-fungal-mitogenome-eval --out-base SRR5801933
slim_fastg.py -F fungus_mt `ls GetOrganelle-fungal-mitogenome/auto--SRR5804119-mitogenome/*.fasta | head -n1` --out-dir GetOrganelle-fungal-mitogenome-eval --out-base SRR5804119
slim_fastg.py -F fungus_mt `ls GetOrganelle-fungal-mitogenome/auto--SRR5804157-mitogenome/*.fasta | head -n1` --out-dir GetOrganelle-fungal-mitogenome-eval --out-base SRR5804157
slim_fastg.py -F fungus_mt `ls GetOrganelle-fungal-mitogenome/auto--SRR5802411-mitogenome/*.fasta | head -n1` --out-dir GetOrganelle-fungal-mitogenome-eval --out-base SRR5802411
slim_fastg.py -F fungus_mt `ls GetOrganelle-fungal-mitogenome/auto--SRR5804131-mitogenome/*.fasta | head -n1` --out-dir GetOrganelle-fungal-mitogenome-eval --out-base SRR5804131
slim_fastg.py -F fungus_mt `ls GetOrganelle-fungal-mitogenome/auto--SRR5765278-mitogenome/*.fasta | head -n1` --out-dir GetOrganelle-fungal-mitogenome-eval --out-base SRR5765278
slim_fastg.py -F fungus_mt `ls GetOrganelle-fungal-mitogenome/auto--SRR5804153-mitogenome/*.fasta | head -n1` --out-dir GetOrganelle-fungal-mitogenome-eval --out-base SRR5804153
slim_fastg.py -F fungus_mt `ls GetOrganelle-fungal-mitogenome/auto--SRR5801932-mitogenome/*.fasta | head -n1` --out-dir GetOrganelle-fungal-mitogenome-eval --out-base SRR5801932
slim_fastg.py -F fungus_mt `ls GetOrganelle-fungal-mitogenome/auto--SRR5804018-mitogenome/*.fasta | head -n1` --out-dir GetOrganelle-fungal-mitogenome-eval --out-base SRR5804018
slim_fastg.py -F fungus_mt `ls GetOrganelle-fungal-mitogenome/auto--SRR5802125-mitogenome/*.fasta | head -n1` --out-dir GetOrganelle-fungal-mitogenome-eval --out-base SRR5802125
slim_fastg.py -F fungus_mt `ls GetOrganelle-fungal-mitogenome/auto--SRR5804118-mitogenome/*.fasta | head -n1` --out-dir GetOrganelle-fungal-mitogenome-eval --out-base SRR5804118
slim_fastg.py -F fungus_mt `ls GetOrganelle-fungal-mitogenome/auto--SRR5804130-mitogenome/*.fasta | head -n1` --out-dir GetOrganelle-fungal-mitogenome-eval --out-base SRR5804130

# only keep the label files
rm GetOrganelle-fungal-mitogenome-eval/*.fasta

# summary
python << END
import os
samples = [x for x in os.listdir("GetOrganelle-fungal-mitogenome-eval") if x.endswith(".csv")]
with open("GetOrganelle-fungal-mitogenome-eval/summary.tab", "w") as output_h:
    for s in samples:
        with open(os.path.join("GetOrganelle-fungal-mitogenome-eval", s)) as input_h:
            head = input_h.readline()
            genes = []
            for line in input_h:
                if line.strip():
                    genes.extend(line.split("\t")[2].split(","))
            genes = sorted(set(genes))
            output_h.write(s.replace(".csv", "") + "\t" + str(len(genes)) + "\t" + ",".join(genes) + "\n")
END
