mkdir NOVOPlasty-fungal-mitogenome-eval

# NOVOPlasty is poorly designed to generate different sequences with identical names
echo "import sys; context = open(sys.argv[1]).readlines(); open(sys.argv[1], 'w').writelines([l.strip() + '__' + str(go) + '\n'  if l.startswith('>') else l for go, l in enumerate(context)]) " > standardize_fas_names.py

# generate label tabel, cat all generated output
cat NOVOPlasty-fungal-mitogenome/K39/SRR5803930-fungal-mitogenome/*fasta > NOVOPlasty-fungal-mitogenome/K39/SRR5803930-fungal-mitogenome/temp.fa ; if [ -n "`ls NOVOPlasty-fungal-mitogenome/K39/SRR5803930-fungal-mitogenome/temp.fa`" ]; then python standardize_fas_names.py NOVOPlasty-fungal-mitogenome/K39/SRR5803930-fungal-mitogenome/temp.fa ; slim_fastg.py -F fungus_mt `ls NOVOPlasty-fungal-mitogenome/K39/SRR5803930-fungal-mitogenome/temp.fa | head -n1` --out-dir NOVOPlasty-fungal-mitogenome-eval --out-base SRR5803930 ; rm NOVOPlasty-fungal-mitogenome/K39/SRR5803930-fungal-mitogenome/temp.fa ; fi; 
cat NOVOPlasty-fungal-mitogenome/K39/SRR5804139-fungal-mitogenome/*fasta > NOVOPlasty-fungal-mitogenome/K39/SRR5804139-fungal-mitogenome/temp.fa ; if [ -n "`ls NOVOPlasty-fungal-mitogenome/K39/SRR5804139-fungal-mitogenome/temp.fa`" ]; then python standardize_fas_names.py NOVOPlasty-fungal-mitogenome/K39/SRR5804139-fungal-mitogenome/temp.fa ; slim_fastg.py -F fungus_mt `ls NOVOPlasty-fungal-mitogenome/K39/SRR5804139-fungal-mitogenome/temp.fa | head -n1` --out-dir NOVOPlasty-fungal-mitogenome-eval --out-base SRR5804139 ; rm NOVOPlasty-fungal-mitogenome/K39/SRR5804139-fungal-mitogenome/temp.fa ; fi; 
cat NOVOPlasty-fungal-mitogenome/K39/SRR5802419-fungal-mitogenome/*fasta > NOVOPlasty-fungal-mitogenome/K39/SRR5802419-fungal-mitogenome/temp.fa ; if [ -n "`ls NOVOPlasty-fungal-mitogenome/K39/SRR5802419-fungal-mitogenome/temp.fa`" ]; then python standardize_fas_names.py NOVOPlasty-fungal-mitogenome/K39/SRR5802419-fungal-mitogenome/temp.fa ; slim_fastg.py -F fungus_mt `ls NOVOPlasty-fungal-mitogenome/K39/SRR5802419-fungal-mitogenome/temp.fa | head -n1` --out-dir NOVOPlasty-fungal-mitogenome-eval --out-base SRR5802419 ; rm NOVOPlasty-fungal-mitogenome/K39/SRR5802419-fungal-mitogenome/temp.fa ; fi; 
cat NOVOPlasty-fungal-mitogenome/K39/SRR5804120-fungal-mitogenome/*fasta > NOVOPlasty-fungal-mitogenome/K39/SRR5804120-fungal-mitogenome/temp.fa ; if [ -n "`ls NOVOPlasty-fungal-mitogenome/K39/SRR5804120-fungal-mitogenome/temp.fa`" ]; then python standardize_fas_names.py NOVOPlasty-fungal-mitogenome/K39/SRR5804120-fungal-mitogenome/temp.fa ; slim_fastg.py -F fungus_mt `ls NOVOPlasty-fungal-mitogenome/K39/SRR5804120-fungal-mitogenome/temp.fa | head -n1` --out-dir NOVOPlasty-fungal-mitogenome-eval --out-base SRR5804120 ; rm NOVOPlasty-fungal-mitogenome/K39/SRR5804120-fungal-mitogenome/temp.fa ; fi; 
cat NOVOPlasty-fungal-mitogenome/K39/SRR5804020-fungal-mitogenome/*fasta > NOVOPlasty-fungal-mitogenome/K39/SRR5804020-fungal-mitogenome/temp.fa ; if [ -n "`ls NOVOPlasty-fungal-mitogenome/K39/SRR5804020-fungal-mitogenome/temp.fa`" ]; then python standardize_fas_names.py NOVOPlasty-fungal-mitogenome/K39/SRR5804020-fungal-mitogenome/temp.fa ; slim_fastg.py -F fungus_mt `ls NOVOPlasty-fungal-mitogenome/K39/SRR5804020-fungal-mitogenome/temp.fa | head -n1` --out-dir NOVOPlasty-fungal-mitogenome-eval --out-base SRR5804020 ; rm NOVOPlasty-fungal-mitogenome/K39/SRR5804020-fungal-mitogenome/temp.fa ; fi; 
cat NOVOPlasty-fungal-mitogenome/K39/SRR5803905-fungal-mitogenome/*fasta > NOVOPlasty-fungal-mitogenome/K39/SRR5803905-fungal-mitogenome/temp.fa ; if [ -n "`ls NOVOPlasty-fungal-mitogenome/K39/SRR5803905-fungal-mitogenome/temp.fa`" ]; then python standardize_fas_names.py NOVOPlasty-fungal-mitogenome/K39/SRR5803905-fungal-mitogenome/temp.fa ; slim_fastg.py -F fungus_mt `ls NOVOPlasty-fungal-mitogenome/K39/SRR5803905-fungal-mitogenome/temp.fa | head -n1` --out-dir NOVOPlasty-fungal-mitogenome-eval --out-base SRR5803905 ; rm NOVOPlasty-fungal-mitogenome/K39/SRR5803905-fungal-mitogenome/temp.fa ; fi; 
cat NOVOPlasty-fungal-mitogenome/K39/SRR5764827-fungal-mitogenome/*fasta > NOVOPlasty-fungal-mitogenome/K39/SRR5764827-fungal-mitogenome/temp.fa ; if [ -n "`ls NOVOPlasty-fungal-mitogenome/K39/SRR5764827-fungal-mitogenome/temp.fa`" ]; then python standardize_fas_names.py NOVOPlasty-fungal-mitogenome/K39/SRR5764827-fungal-mitogenome/temp.fa ; slim_fastg.py -F fungus_mt `ls NOVOPlasty-fungal-mitogenome/K39/SRR5764827-fungal-mitogenome/temp.fa | head -n1` --out-dir NOVOPlasty-fungal-mitogenome-eval --out-base SRR5764827 ; rm NOVOPlasty-fungal-mitogenome/K39/SRR5764827-fungal-mitogenome/temp.fa ; fi; 
cat NOVOPlasty-fungal-mitogenome/K39/SRR5804147-fungal-mitogenome/*fasta > NOVOPlasty-fungal-mitogenome/K39/SRR5804147-fungal-mitogenome/temp.fa ; if [ -n "`ls NOVOPlasty-fungal-mitogenome/K39/SRR5804147-fungal-mitogenome/temp.fa`" ]; then python standardize_fas_names.py NOVOPlasty-fungal-mitogenome/K39/SRR5804147-fungal-mitogenome/temp.fa ; slim_fastg.py -F fungus_mt `ls NOVOPlasty-fungal-mitogenome/K39/SRR5804147-fungal-mitogenome/temp.fa | head -n1` --out-dir NOVOPlasty-fungal-mitogenome-eval --out-base SRR5804147 ; rm NOVOPlasty-fungal-mitogenome/K39/SRR5804147-fungal-mitogenome/temp.fa ; fi; 
cat NOVOPlasty-fungal-mitogenome/K39/SRR5804015-fungal-mitogenome/*fasta > NOVOPlasty-fungal-mitogenome/K39/SRR5804015-fungal-mitogenome/temp.fa ; if [ -n "`ls NOVOPlasty-fungal-mitogenome/K39/SRR5804015-fungal-mitogenome/temp.fa`" ]; then python standardize_fas_names.py NOVOPlasty-fungal-mitogenome/K39/SRR5804015-fungal-mitogenome/temp.fa ; slim_fastg.py -F fungus_mt `ls NOVOPlasty-fungal-mitogenome/K39/SRR5804015-fungal-mitogenome/temp.fa | head -n1` --out-dir NOVOPlasty-fungal-mitogenome-eval --out-base SRR5804015 ; rm NOVOPlasty-fungal-mitogenome/K39/SRR5804015-fungal-mitogenome/temp.fa ; fi; 
cat NOVOPlasty-fungal-mitogenome/K39/SRR5804115-fungal-mitogenome/*fasta > NOVOPlasty-fungal-mitogenome/K39/SRR5804115-fungal-mitogenome/temp.fa ; if [ -n "`ls NOVOPlasty-fungal-mitogenome/K39/SRR5804115-fungal-mitogenome/temp.fa`" ]; then python standardize_fas_names.py NOVOPlasty-fungal-mitogenome/K39/SRR5804115-fungal-mitogenome/temp.fa ; slim_fastg.py -F fungus_mt `ls NOVOPlasty-fungal-mitogenome/K39/SRR5804115-fungal-mitogenome/temp.fa | head -n1` --out-dir NOVOPlasty-fungal-mitogenome-eval --out-base SRR5804115 ; rm NOVOPlasty-fungal-mitogenome/K39/SRR5804115-fungal-mitogenome/temp.fa ; fi; 
cat NOVOPlasty-fungal-mitogenome/K39/SRR5754079-fungal-mitogenome/*fasta > NOVOPlasty-fungal-mitogenome/K39/SRR5754079-fungal-mitogenome/temp.fa ; if [ -n "`ls NOVOPlasty-fungal-mitogenome/K39/SRR5754079-fungal-mitogenome/temp.fa`" ]; then python standardize_fas_names.py NOVOPlasty-fungal-mitogenome/K39/SRR5754079-fungal-mitogenome/temp.fa ; slim_fastg.py -F fungus_mt `ls NOVOPlasty-fungal-mitogenome/K39/SRR5754079-fungal-mitogenome/temp.fa | head -n1` --out-dir NOVOPlasty-fungal-mitogenome-eval --out-base SRR5754079 ; rm NOVOPlasty-fungal-mitogenome/K39/SRR5754079-fungal-mitogenome/temp.fa ; fi; 
cat NOVOPlasty-fungal-mitogenome/K39/SRR5803928-fungal-mitogenome/*fasta > NOVOPlasty-fungal-mitogenome/K39/SRR5803928-fungal-mitogenome/temp.fa ; if [ -n "`ls NOVOPlasty-fungal-mitogenome/K39/SRR5803928-fungal-mitogenome/temp.fa`" ]; then python standardize_fas_names.py NOVOPlasty-fungal-mitogenome/K39/SRR5803928-fungal-mitogenome/temp.fa ; slim_fastg.py -F fungus_mt `ls NOVOPlasty-fungal-mitogenome/K39/SRR5803928-fungal-mitogenome/temp.fa | head -n1` --out-dir NOVOPlasty-fungal-mitogenome-eval --out-base SRR5803928 ; rm NOVOPlasty-fungal-mitogenome/K39/SRR5803928-fungal-mitogenome/temp.fa ; fi; 
cat NOVOPlasty-fungal-mitogenome/K39/SRR5765276-fungal-mitogenome/*fasta > NOVOPlasty-fungal-mitogenome/K39/SRR5765276-fungal-mitogenome/temp.fa ; if [ -n "`ls NOVOPlasty-fungal-mitogenome/K39/SRR5765276-fungal-mitogenome/temp.fa`" ]; then python standardize_fas_names.py NOVOPlasty-fungal-mitogenome/K39/SRR5765276-fungal-mitogenome/temp.fa ; slim_fastg.py -F fungus_mt `ls NOVOPlasty-fungal-mitogenome/K39/SRR5765276-fungal-mitogenome/temp.fa | head -n1` --out-dir NOVOPlasty-fungal-mitogenome-eval --out-base SRR5765276 ; rm NOVOPlasty-fungal-mitogenome/K39/SRR5765276-fungal-mitogenome/temp.fa ; fi; 
cat NOVOPlasty-fungal-mitogenome/K39/SRR5804127-fungal-mitogenome/*fasta > NOVOPlasty-fungal-mitogenome/K39/SRR5804127-fungal-mitogenome/temp.fa ; if [ -n "`ls NOVOPlasty-fungal-mitogenome/K39/SRR5804127-fungal-mitogenome/temp.fa`" ]; then python standardize_fas_names.py NOVOPlasty-fungal-mitogenome/K39/SRR5804127-fungal-mitogenome/temp.fa ; slim_fastg.py -F fungus_mt `ls NOVOPlasty-fungal-mitogenome/K39/SRR5804127-fungal-mitogenome/temp.fa | head -n1` --out-dir NOVOPlasty-fungal-mitogenome-eval --out-base SRR5804127 ; rm NOVOPlasty-fungal-mitogenome/K39/SRR5804127-fungal-mitogenome/temp.fa ; fi; 
cat NOVOPlasty-fungal-mitogenome/K39/SRR5801943-fungal-mitogenome/*fasta > NOVOPlasty-fungal-mitogenome/K39/SRR5801943-fungal-mitogenome/temp.fa ; if [ -n "`ls NOVOPlasty-fungal-mitogenome/K39/SRR5801943-fungal-mitogenome/temp.fa`" ]; then python standardize_fas_names.py NOVOPlasty-fungal-mitogenome/K39/SRR5801943-fungal-mitogenome/temp.fa ; slim_fastg.py -F fungus_mt `ls NOVOPlasty-fungal-mitogenome/K39/SRR5801943-fungal-mitogenome/temp.fa | head -n1` --out-dir NOVOPlasty-fungal-mitogenome-eval --out-base SRR5801943 ; rm NOVOPlasty-fungal-mitogenome/K39/SRR5801943-fungal-mitogenome/temp.fa ; fi; 
cat NOVOPlasty-fungal-mitogenome/K39/SRR5802181-fungal-mitogenome/*fasta > NOVOPlasty-fungal-mitogenome/K39/SRR5802181-fungal-mitogenome/temp.fa ; if [ -n "`ls NOVOPlasty-fungal-mitogenome/K39/SRR5802181-fungal-mitogenome/temp.fa`" ]; then python standardize_fas_names.py NOVOPlasty-fungal-mitogenome/K39/SRR5802181-fungal-mitogenome/temp.fa ; slim_fastg.py -F fungus_mt `ls NOVOPlasty-fungal-mitogenome/K39/SRR5802181-fungal-mitogenome/temp.fa | head -n1` --out-dir NOVOPlasty-fungal-mitogenome-eval --out-base SRR5802181 ; rm NOVOPlasty-fungal-mitogenome/K39/SRR5802181-fungal-mitogenome/temp.fa ; fi; 
cat NOVOPlasty-fungal-mitogenome/K39/SRR5803933-fungal-mitogenome/*fasta > NOVOPlasty-fungal-mitogenome/K39/SRR5803933-fungal-mitogenome/temp.fa ; if [ -n "`ls NOVOPlasty-fungal-mitogenome/K39/SRR5803933-fungal-mitogenome/temp.fa`" ]; then python standardize_fas_names.py NOVOPlasty-fungal-mitogenome/K39/SRR5803933-fungal-mitogenome/temp.fa ; slim_fastg.py -F fungus_mt `ls NOVOPlasty-fungal-mitogenome/K39/SRR5803933-fungal-mitogenome/temp.fa | head -n1` --out-dir NOVOPlasty-fungal-mitogenome-eval --out-base SRR5803933 ; rm NOVOPlasty-fungal-mitogenome/K39/SRR5803933-fungal-mitogenome/temp.fa ; fi; 
cat NOVOPlasty-fungal-mitogenome/K39/SRR5801947-fungal-mitogenome/*fasta > NOVOPlasty-fungal-mitogenome/K39/SRR5801947-fungal-mitogenome/temp.fa ; if [ -n "`ls NOVOPlasty-fungal-mitogenome/K39/SRR5801947-fungal-mitogenome/temp.fa`" ]; then python standardize_fas_names.py NOVOPlasty-fungal-mitogenome/K39/SRR5801947-fungal-mitogenome/temp.fa ; slim_fastg.py -F fungus_mt `ls NOVOPlasty-fungal-mitogenome/K39/SRR5801947-fungal-mitogenome/temp.fa | head -n1` --out-dir NOVOPlasty-fungal-mitogenome-eval --out-base SRR5801947 ; rm NOVOPlasty-fungal-mitogenome/K39/SRR5801947-fungal-mitogenome/temp.fa ; fi; 
cat NOVOPlasty-fungal-mitogenome/K39/SRR5804112-fungal-mitogenome/*fasta > NOVOPlasty-fungal-mitogenome/K39/SRR5804112-fungal-mitogenome/temp.fa ; if [ -n "`ls NOVOPlasty-fungal-mitogenome/K39/SRR5804112-fungal-mitogenome/temp.fa`" ]; then python standardize_fas_names.py NOVOPlasty-fungal-mitogenome/K39/SRR5804112-fungal-mitogenome/temp.fa ; slim_fastg.py -F fungus_mt `ls NOVOPlasty-fungal-mitogenome/K39/SRR5804112-fungal-mitogenome/temp.fa | head -n1` --out-dir NOVOPlasty-fungal-mitogenome-eval --out-base SRR5804112 ; rm NOVOPlasty-fungal-mitogenome/K39/SRR5804112-fungal-mitogenome/temp.fa ; fi; 
cat NOVOPlasty-fungal-mitogenome/K39/SRR5804145-fungal-mitogenome/*fasta > NOVOPlasty-fungal-mitogenome/K39/SRR5804145-fungal-mitogenome/temp.fa ; if [ -n "`ls NOVOPlasty-fungal-mitogenome/K39/SRR5804145-fungal-mitogenome/temp.fa`" ]; then python standardize_fas_names.py NOVOPlasty-fungal-mitogenome/K39/SRR5804145-fungal-mitogenome/temp.fa ; slim_fastg.py -F fungus_mt `ls NOVOPlasty-fungal-mitogenome/K39/SRR5804145-fungal-mitogenome/temp.fa | head -n1` --out-dir NOVOPlasty-fungal-mitogenome-eval --out-base SRR5804145 ; rm NOVOPlasty-fungal-mitogenome/K39/SRR5804145-fungal-mitogenome/temp.fa ; fi; 
cat NOVOPlasty-fungal-mitogenome/K39/SRR5802698-fungal-mitogenome/*fasta > NOVOPlasty-fungal-mitogenome/K39/SRR5802698-fungal-mitogenome/temp.fa ; if [ -n "`ls NOVOPlasty-fungal-mitogenome/K39/SRR5802698-fungal-mitogenome/temp.fa`" ]; then python standardize_fas_names.py NOVOPlasty-fungal-mitogenome/K39/SRR5802698-fungal-mitogenome/temp.fa ; slim_fastg.py -F fungus_mt `ls NOVOPlasty-fungal-mitogenome/K39/SRR5802698-fungal-mitogenome/temp.fa | head -n1` --out-dir NOVOPlasty-fungal-mitogenome-eval --out-base SRR5802698 ; rm NOVOPlasty-fungal-mitogenome/K39/SRR5802698-fungal-mitogenome/temp.fa ; fi; 
cat NOVOPlasty-fungal-mitogenome/K39/SRR5804017-fungal-mitogenome/*fasta > NOVOPlasty-fungal-mitogenome/K39/SRR5804017-fungal-mitogenome/temp.fa ; if [ -n "`ls NOVOPlasty-fungal-mitogenome/K39/SRR5804017-fungal-mitogenome/temp.fa`" ]; then python standardize_fas_names.py NOVOPlasty-fungal-mitogenome/K39/SRR5804017-fungal-mitogenome/temp.fa ; slim_fastg.py -F fungus_mt `ls NOVOPlasty-fungal-mitogenome/K39/SRR5804017-fungal-mitogenome/temp.fa | head -n1` --out-dir NOVOPlasty-fungal-mitogenome-eval --out-base SRR5804017 ; rm NOVOPlasty-fungal-mitogenome/K39/SRR5804017-fungal-mitogenome/temp.fa ; fi; 
cat NOVOPlasty-fungal-mitogenome/K39/SRR5804159-fungal-mitogenome/*fasta > NOVOPlasty-fungal-mitogenome/K39/SRR5804159-fungal-mitogenome/temp.fa ; if [ -n "`ls NOVOPlasty-fungal-mitogenome/K39/SRR5804159-fungal-mitogenome/temp.fa`" ]; then python standardize_fas_names.py NOVOPlasty-fungal-mitogenome/K39/SRR5804159-fungal-mitogenome/temp.fa ; slim_fastg.py -F fungus_mt `ls NOVOPlasty-fungal-mitogenome/K39/SRR5804159-fungal-mitogenome/temp.fa | head -n1` --out-dir NOVOPlasty-fungal-mitogenome-eval --out-base SRR5804159 ; rm NOVOPlasty-fungal-mitogenome/K39/SRR5804159-fungal-mitogenome/temp.fa ; fi; 
cat NOVOPlasty-fungal-mitogenome/K39/SRR5804023-fungal-mitogenome/*fasta > NOVOPlasty-fungal-mitogenome/K39/SRR5804023-fungal-mitogenome/temp.fa ; if [ -n "`ls NOVOPlasty-fungal-mitogenome/K39/SRR5804023-fungal-mitogenome/temp.fa`" ]; then python standardize_fas_names.py NOVOPlasty-fungal-mitogenome/K39/SRR5804023-fungal-mitogenome/temp.fa ; slim_fastg.py -F fungus_mt `ls NOVOPlasty-fungal-mitogenome/K39/SRR5804023-fungal-mitogenome/temp.fa | head -n1` --out-dir NOVOPlasty-fungal-mitogenome-eval --out-base SRR5804023 ; rm NOVOPlasty-fungal-mitogenome/K39/SRR5804023-fungal-mitogenome/temp.fa ; fi; 
cat NOVOPlasty-fungal-mitogenome/K39/SRR5803923-fungal-mitogenome/*fasta > NOVOPlasty-fungal-mitogenome/K39/SRR5803923-fungal-mitogenome/temp.fa ; if [ -n "`ls NOVOPlasty-fungal-mitogenome/K39/SRR5803923-fungal-mitogenome/temp.fa`" ]; then python standardize_fas_names.py NOVOPlasty-fungal-mitogenome/K39/SRR5803923-fungal-mitogenome/temp.fa ; slim_fastg.py -F fungus_mt `ls NOVOPlasty-fungal-mitogenome/K39/SRR5803923-fungal-mitogenome/temp.fa | head -n1` --out-dir NOVOPlasty-fungal-mitogenome-eval --out-base SRR5803923 ; rm NOVOPlasty-fungal-mitogenome/K39/SRR5803923-fungal-mitogenome/temp.fa ; fi; 
cat NOVOPlasty-fungal-mitogenome/K39/SRR5803890-fungal-mitogenome/*fasta > NOVOPlasty-fungal-mitogenome/K39/SRR5803890-fungal-mitogenome/temp.fa ; if [ -n "`ls NOVOPlasty-fungal-mitogenome/K39/SRR5803890-fungal-mitogenome/temp.fa`" ]; then python standardize_fas_names.py NOVOPlasty-fungal-mitogenome/K39/SRR5803890-fungal-mitogenome/temp.fa ; slim_fastg.py -F fungus_mt `ls NOVOPlasty-fungal-mitogenome/K39/SRR5803890-fungal-mitogenome/temp.fa | head -n1` --out-dir NOVOPlasty-fungal-mitogenome-eval --out-base SRR5803890 ; rm NOVOPlasty-fungal-mitogenome/K39/SRR5803890-fungal-mitogenome/temp.fa ; fi; 
cat NOVOPlasty-fungal-mitogenome/K39/SRR5764829-fungal-mitogenome/*fasta > NOVOPlasty-fungal-mitogenome/K39/SRR5764829-fungal-mitogenome/temp.fa ; if [ -n "`ls NOVOPlasty-fungal-mitogenome/K39/SRR5764829-fungal-mitogenome/temp.fa`" ]; then python standardize_fas_names.py NOVOPlasty-fungal-mitogenome/K39/SRR5764829-fungal-mitogenome/temp.fa ; slim_fastg.py -F fungus_mt `ls NOVOPlasty-fungal-mitogenome/K39/SRR5764829-fungal-mitogenome/temp.fa | head -n1` --out-dir NOVOPlasty-fungal-mitogenome-eval --out-base SRR5764829 ; rm NOVOPlasty-fungal-mitogenome/K39/SRR5764829-fungal-mitogenome/temp.fa ; fi; 
cat NOVOPlasty-fungal-mitogenome/K39/SRR5803926-fungal-mitogenome/*fasta > NOVOPlasty-fungal-mitogenome/K39/SRR5803926-fungal-mitogenome/temp.fa ; if [ -n "`ls NOVOPlasty-fungal-mitogenome/K39/SRR5803926-fungal-mitogenome/temp.fa`" ]; then python standardize_fas_names.py NOVOPlasty-fungal-mitogenome/K39/SRR5803926-fungal-mitogenome/temp.fa ; slim_fastg.py -F fungus_mt `ls NOVOPlasty-fungal-mitogenome/K39/SRR5803926-fungal-mitogenome/temp.fa | head -n1` --out-dir NOVOPlasty-fungal-mitogenome-eval --out-base SRR5803926 ; rm NOVOPlasty-fungal-mitogenome/K39/SRR5803926-fungal-mitogenome/temp.fa ; fi; 
cat NOVOPlasty-fungal-mitogenome/K39/SRR5764835-fungal-mitogenome/*fasta > NOVOPlasty-fungal-mitogenome/K39/SRR5764835-fungal-mitogenome/temp.fa ; if [ -n "`ls NOVOPlasty-fungal-mitogenome/K39/SRR5764835-fungal-mitogenome/temp.fa`" ]; then python standardize_fas_names.py NOVOPlasty-fungal-mitogenome/K39/SRR5764835-fungal-mitogenome/temp.fa ; slim_fastg.py -F fungus_mt `ls NOVOPlasty-fungal-mitogenome/K39/SRR5764835-fungal-mitogenome/temp.fa | head -n1` --out-dir NOVOPlasty-fungal-mitogenome-eval --out-base SRR5764835 ; rm NOVOPlasty-fungal-mitogenome/K39/SRR5764835-fungal-mitogenome/temp.fa ; fi; 
cat NOVOPlasty-fungal-mitogenome/K39/SRR5802413-fungal-mitogenome/*fasta > NOVOPlasty-fungal-mitogenome/K39/SRR5802413-fungal-mitogenome/temp.fa ; if [ -n "`ls NOVOPlasty-fungal-mitogenome/K39/SRR5802413-fungal-mitogenome/temp.fa`" ]; then python standardize_fas_names.py NOVOPlasty-fungal-mitogenome/K39/SRR5802413-fungal-mitogenome/temp.fa ; slim_fastg.py -F fungus_mt `ls NOVOPlasty-fungal-mitogenome/K39/SRR5802413-fungal-mitogenome/temp.fa | head -n1` --out-dir NOVOPlasty-fungal-mitogenome-eval --out-base SRR5802413 ; rm NOVOPlasty-fungal-mitogenome/K39/SRR5802413-fungal-mitogenome/temp.fa ; fi; 
cat NOVOPlasty-fungal-mitogenome/K39/SRR5801934-fungal-mitogenome/*fasta > NOVOPlasty-fungal-mitogenome/K39/SRR5801934-fungal-mitogenome/temp.fa ; if [ -n "`ls NOVOPlasty-fungal-mitogenome/K39/SRR5801934-fungal-mitogenome/temp.fa`" ]; then python standardize_fas_names.py NOVOPlasty-fungal-mitogenome/K39/SRR5801934-fungal-mitogenome/temp.fa ; slim_fastg.py -F fungus_mt `ls NOVOPlasty-fungal-mitogenome/K39/SRR5801934-fungal-mitogenome/temp.fa | head -n1` --out-dir NOVOPlasty-fungal-mitogenome-eval --out-base SRR5801934 ; rm NOVOPlasty-fungal-mitogenome/K39/SRR5801934-fungal-mitogenome/temp.fa ; fi; 
cat NOVOPlasty-fungal-mitogenome/K39/SRR5803916-fungal-mitogenome/*fasta > NOVOPlasty-fungal-mitogenome/K39/SRR5803916-fungal-mitogenome/temp.fa ; if [ -n "`ls NOVOPlasty-fungal-mitogenome/K39/SRR5803916-fungal-mitogenome/temp.fa`" ]; then python standardize_fas_names.py NOVOPlasty-fungal-mitogenome/K39/SRR5803916-fungal-mitogenome/temp.fa ; slim_fastg.py -F fungus_mt `ls NOVOPlasty-fungal-mitogenome/K39/SRR5803916-fungal-mitogenome/temp.fa | head -n1` --out-dir NOVOPlasty-fungal-mitogenome-eval --out-base SRR5803916 ; rm NOVOPlasty-fungal-mitogenome/K39/SRR5803916-fungal-mitogenome/temp.fa ; fi; 
cat NOVOPlasty-fungal-mitogenome/K39/SRR5803922-fungal-mitogenome/*fasta > NOVOPlasty-fungal-mitogenome/K39/SRR5803922-fungal-mitogenome/temp.fa ; if [ -n "`ls NOVOPlasty-fungal-mitogenome/K39/SRR5803922-fungal-mitogenome/temp.fa`" ]; then python standardize_fas_names.py NOVOPlasty-fungal-mitogenome/K39/SRR5803922-fungal-mitogenome/temp.fa ; slim_fastg.py -F fungus_mt `ls NOVOPlasty-fungal-mitogenome/K39/SRR5803922-fungal-mitogenome/temp.fa | head -n1` --out-dir NOVOPlasty-fungal-mitogenome-eval --out-base SRR5803922 ; rm NOVOPlasty-fungal-mitogenome/K39/SRR5803922-fungal-mitogenome/temp.fa ; fi; 
cat NOVOPlasty-fungal-mitogenome/K39/SRR5804137-fungal-mitogenome/*fasta > NOVOPlasty-fungal-mitogenome/K39/SRR5804137-fungal-mitogenome/temp.fa ; if [ -n "`ls NOVOPlasty-fungal-mitogenome/K39/SRR5804137-fungal-mitogenome/temp.fa`" ]; then python standardize_fas_names.py NOVOPlasty-fungal-mitogenome/K39/SRR5804137-fungal-mitogenome/temp.fa ; slim_fastg.py -F fungus_mt `ls NOVOPlasty-fungal-mitogenome/K39/SRR5804137-fungal-mitogenome/temp.fa | head -n1` --out-dir NOVOPlasty-fungal-mitogenome-eval --out-base SRR5804137 ; rm NOVOPlasty-fungal-mitogenome/K39/SRR5804137-fungal-mitogenome/temp.fa ; fi; 
cat NOVOPlasty-fungal-mitogenome/K39/SRR5801929-fungal-mitogenome/*fasta > NOVOPlasty-fungal-mitogenome/K39/SRR5801929-fungal-mitogenome/temp.fa ; if [ -n "`ls NOVOPlasty-fungal-mitogenome/K39/SRR5801929-fungal-mitogenome/temp.fa`" ]; then python standardize_fas_names.py NOVOPlasty-fungal-mitogenome/K39/SRR5801929-fungal-mitogenome/temp.fa ; slim_fastg.py -F fungus_mt `ls NOVOPlasty-fungal-mitogenome/K39/SRR5801929-fungal-mitogenome/temp.fa | head -n1` --out-dir NOVOPlasty-fungal-mitogenome-eval --out-base SRR5801929 ; rm NOVOPlasty-fungal-mitogenome/K39/SRR5801929-fungal-mitogenome/temp.fa ; fi; 
cat NOVOPlasty-fungal-mitogenome/K39/SRR5803913-fungal-mitogenome/*fasta > NOVOPlasty-fungal-mitogenome/K39/SRR5803913-fungal-mitogenome/temp.fa ; if [ -n "`ls NOVOPlasty-fungal-mitogenome/K39/SRR5803913-fungal-mitogenome/temp.fa`" ]; then python standardize_fas_names.py NOVOPlasty-fungal-mitogenome/K39/SRR5803913-fungal-mitogenome/temp.fa ; slim_fastg.py -F fungus_mt `ls NOVOPlasty-fungal-mitogenome/K39/SRR5803913-fungal-mitogenome/temp.fa | head -n1` --out-dir NOVOPlasty-fungal-mitogenome-eval --out-base SRR5803913 ; rm NOVOPlasty-fungal-mitogenome/K39/SRR5803913-fungal-mitogenome/temp.fa ; fi; 
cat NOVOPlasty-fungal-mitogenome/K39/SRR5801935-fungal-mitogenome/*fasta > NOVOPlasty-fungal-mitogenome/K39/SRR5801935-fungal-mitogenome/temp.fa ; if [ -n "`ls NOVOPlasty-fungal-mitogenome/K39/SRR5801935-fungal-mitogenome/temp.fa`" ]; then python standardize_fas_names.py NOVOPlasty-fungal-mitogenome/K39/SRR5801935-fungal-mitogenome/temp.fa ; slim_fastg.py -F fungus_mt `ls NOVOPlasty-fungal-mitogenome/K39/SRR5801935-fungal-mitogenome/temp.fa | head -n1` --out-dir NOVOPlasty-fungal-mitogenome-eval --out-base SRR5801935 ; rm NOVOPlasty-fungal-mitogenome/K39/SRR5801935-fungal-mitogenome/temp.fa ; fi; 
cat NOVOPlasty-fungal-mitogenome/K39/SRR5803909-fungal-mitogenome/*fasta > NOVOPlasty-fungal-mitogenome/K39/SRR5803909-fungal-mitogenome/temp.fa ; if [ -n "`ls NOVOPlasty-fungal-mitogenome/K39/SRR5803909-fungal-mitogenome/temp.fa`" ]; then python standardize_fas_names.py NOVOPlasty-fungal-mitogenome/K39/SRR5803909-fungal-mitogenome/temp.fa ; slim_fastg.py -F fungus_mt `ls NOVOPlasty-fungal-mitogenome/K39/SRR5803909-fungal-mitogenome/temp.fa | head -n1` --out-dir NOVOPlasty-fungal-mitogenome-eval --out-base SRR5803909 ; rm NOVOPlasty-fungal-mitogenome/K39/SRR5803909-fungal-mitogenome/temp.fa ; fi; 
cat NOVOPlasty-fungal-mitogenome/K39/SRR5801933-fungal-mitogenome/*fasta > NOVOPlasty-fungal-mitogenome/K39/SRR5801933-fungal-mitogenome/temp.fa ; if [ -n "`ls NOVOPlasty-fungal-mitogenome/K39/SRR5801933-fungal-mitogenome/temp.fa`" ]; then python standardize_fas_names.py NOVOPlasty-fungal-mitogenome/K39/SRR5801933-fungal-mitogenome/temp.fa ; slim_fastg.py -F fungus_mt `ls NOVOPlasty-fungal-mitogenome/K39/SRR5801933-fungal-mitogenome/temp.fa | head -n1` --out-dir NOVOPlasty-fungal-mitogenome-eval --out-base SRR5801933 ; rm NOVOPlasty-fungal-mitogenome/K39/SRR5801933-fungal-mitogenome/temp.fa ; fi; 
cat NOVOPlasty-fungal-mitogenome/K39/SRR5804119-fungal-mitogenome/*fasta > NOVOPlasty-fungal-mitogenome/K39/SRR5804119-fungal-mitogenome/temp.fa ; if [ -n "`ls NOVOPlasty-fungal-mitogenome/K39/SRR5804119-fungal-mitogenome/temp.fa`" ]; then python standardize_fas_names.py NOVOPlasty-fungal-mitogenome/K39/SRR5804119-fungal-mitogenome/temp.fa ; slim_fastg.py -F fungus_mt `ls NOVOPlasty-fungal-mitogenome/K39/SRR5804119-fungal-mitogenome/temp.fa | head -n1` --out-dir NOVOPlasty-fungal-mitogenome-eval --out-base SRR5804119 ; rm NOVOPlasty-fungal-mitogenome/K39/SRR5804119-fungal-mitogenome/temp.fa ; fi; 
cat NOVOPlasty-fungal-mitogenome/K39/SRR5804157-fungal-mitogenome/*fasta > NOVOPlasty-fungal-mitogenome/K39/SRR5804157-fungal-mitogenome/temp.fa ; if [ -n "`ls NOVOPlasty-fungal-mitogenome/K39/SRR5804157-fungal-mitogenome/temp.fa`" ]; then python standardize_fas_names.py NOVOPlasty-fungal-mitogenome/K39/SRR5804157-fungal-mitogenome/temp.fa ; slim_fastg.py -F fungus_mt `ls NOVOPlasty-fungal-mitogenome/K39/SRR5804157-fungal-mitogenome/temp.fa | head -n1` --out-dir NOVOPlasty-fungal-mitogenome-eval --out-base SRR5804157 ; rm NOVOPlasty-fungal-mitogenome/K39/SRR5804157-fungal-mitogenome/temp.fa ; fi; 
cat NOVOPlasty-fungal-mitogenome/K39/SRR5802411-fungal-mitogenome/*fasta > NOVOPlasty-fungal-mitogenome/K39/SRR5802411-fungal-mitogenome/temp.fa ; if [ -n "`ls NOVOPlasty-fungal-mitogenome/K39/SRR5802411-fungal-mitogenome/temp.fa`" ]; then python standardize_fas_names.py NOVOPlasty-fungal-mitogenome/K39/SRR5802411-fungal-mitogenome/temp.fa ; slim_fastg.py -F fungus_mt `ls NOVOPlasty-fungal-mitogenome/K39/SRR5802411-fungal-mitogenome/temp.fa | head -n1` --out-dir NOVOPlasty-fungal-mitogenome-eval --out-base SRR5802411 ; rm NOVOPlasty-fungal-mitogenome/K39/SRR5802411-fungal-mitogenome/temp.fa ; fi; 
cat NOVOPlasty-fungal-mitogenome/K39/SRR5804131-fungal-mitogenome/*fasta > NOVOPlasty-fungal-mitogenome/K39/SRR5804131-fungal-mitogenome/temp.fa ; if [ -n "`ls NOVOPlasty-fungal-mitogenome/K39/SRR5804131-fungal-mitogenome/temp.fa`" ]; then python standardize_fas_names.py NOVOPlasty-fungal-mitogenome/K39/SRR5804131-fungal-mitogenome/temp.fa ; slim_fastg.py -F fungus_mt `ls NOVOPlasty-fungal-mitogenome/K39/SRR5804131-fungal-mitogenome/temp.fa | head -n1` --out-dir NOVOPlasty-fungal-mitogenome-eval --out-base SRR5804131 ; rm NOVOPlasty-fungal-mitogenome/K39/SRR5804131-fungal-mitogenome/temp.fa ; fi; 
cat NOVOPlasty-fungal-mitogenome/K39/SRR5765278-fungal-mitogenome/*fasta > NOVOPlasty-fungal-mitogenome/K39/SRR5765278-fungal-mitogenome/temp.fa ; if [ -n "`ls NOVOPlasty-fungal-mitogenome/K39/SRR5765278-fungal-mitogenome/temp.fa`" ]; then python standardize_fas_names.py NOVOPlasty-fungal-mitogenome/K39/SRR5765278-fungal-mitogenome/temp.fa ; slim_fastg.py -F fungus_mt `ls NOVOPlasty-fungal-mitogenome/K39/SRR5765278-fungal-mitogenome/temp.fa | head -n1` --out-dir NOVOPlasty-fungal-mitogenome-eval --out-base SRR5765278 ; rm NOVOPlasty-fungal-mitogenome/K39/SRR5765278-fungal-mitogenome/temp.fa ; fi; 
cat NOVOPlasty-fungal-mitogenome/K39/SRR5804153-fungal-mitogenome/*fasta > NOVOPlasty-fungal-mitogenome/K39/SRR5804153-fungal-mitogenome/temp.fa ; if [ -n "`ls NOVOPlasty-fungal-mitogenome/K39/SRR5804153-fungal-mitogenome/temp.fa`" ]; then python standardize_fas_names.py NOVOPlasty-fungal-mitogenome/K39/SRR5804153-fungal-mitogenome/temp.fa ; slim_fastg.py -F fungus_mt `ls NOVOPlasty-fungal-mitogenome/K39/SRR5804153-fungal-mitogenome/temp.fa | head -n1` --out-dir NOVOPlasty-fungal-mitogenome-eval --out-base SRR5804153 ; rm NOVOPlasty-fungal-mitogenome/K39/SRR5804153-fungal-mitogenome/temp.fa ; fi; 
cat NOVOPlasty-fungal-mitogenome/K39/SRR5801932-fungal-mitogenome/*fasta > NOVOPlasty-fungal-mitogenome/K39/SRR5801932-fungal-mitogenome/temp.fa ; if [ -n "`ls NOVOPlasty-fungal-mitogenome/K39/SRR5801932-fungal-mitogenome/temp.fa`" ]; then python standardize_fas_names.py NOVOPlasty-fungal-mitogenome/K39/SRR5801932-fungal-mitogenome/temp.fa ; slim_fastg.py -F fungus_mt `ls NOVOPlasty-fungal-mitogenome/K39/SRR5801932-fungal-mitogenome/temp.fa | head -n1` --out-dir NOVOPlasty-fungal-mitogenome-eval --out-base SRR5801932 ; rm NOVOPlasty-fungal-mitogenome/K39/SRR5801932-fungal-mitogenome/temp.fa ; fi; 
cat NOVOPlasty-fungal-mitogenome/K39/SRR5804018-fungal-mitogenome/*fasta > NOVOPlasty-fungal-mitogenome/K39/SRR5804018-fungal-mitogenome/temp.fa ; if [ -n "`ls NOVOPlasty-fungal-mitogenome/K39/SRR5804018-fungal-mitogenome/temp.fa`" ]; then python standardize_fas_names.py NOVOPlasty-fungal-mitogenome/K39/SRR5804018-fungal-mitogenome/temp.fa ; slim_fastg.py -F fungus_mt `ls NOVOPlasty-fungal-mitogenome/K39/SRR5804018-fungal-mitogenome/temp.fa | head -n1` --out-dir NOVOPlasty-fungal-mitogenome-eval --out-base SRR5804018 ; rm NOVOPlasty-fungal-mitogenome/K39/SRR5804018-fungal-mitogenome/temp.fa ; fi; 
cat NOVOPlasty-fungal-mitogenome/K39/SRR5802125-fungal-mitogenome/*fasta > NOVOPlasty-fungal-mitogenome/K39/SRR5802125-fungal-mitogenome/temp.fa ; if [ -n "`ls NOVOPlasty-fungal-mitogenome/K39/SRR5802125-fungal-mitogenome/temp.fa`" ]; then python standardize_fas_names.py NOVOPlasty-fungal-mitogenome/K39/SRR5802125-fungal-mitogenome/temp.fa ; slim_fastg.py -F fungus_mt `ls NOVOPlasty-fungal-mitogenome/K39/SRR5802125-fungal-mitogenome/temp.fa | head -n1` --out-dir NOVOPlasty-fungal-mitogenome-eval --out-base SRR5802125 ; rm NOVOPlasty-fungal-mitogenome/K39/SRR5802125-fungal-mitogenome/temp.fa ; fi; 
cat NOVOPlasty-fungal-mitogenome/K39/SRR5804118-fungal-mitogenome/*fasta > NOVOPlasty-fungal-mitogenome/K39/SRR5804118-fungal-mitogenome/temp.fa ; if [ -n "`ls NOVOPlasty-fungal-mitogenome/K39/SRR5804118-fungal-mitogenome/temp.fa`" ]; then python standardize_fas_names.py NOVOPlasty-fungal-mitogenome/K39/SRR5804118-fungal-mitogenome/temp.fa ; slim_fastg.py -F fungus_mt `ls NOVOPlasty-fungal-mitogenome/K39/SRR5804118-fungal-mitogenome/temp.fa | head -n1` --out-dir NOVOPlasty-fungal-mitogenome-eval --out-base SRR5804118 ; rm NOVOPlasty-fungal-mitogenome/K39/SRR5804118-fungal-mitogenome/temp.fa ; fi; 
cat NOVOPlasty-fungal-mitogenome/K39/SRR5804130-fungal-mitogenome/*fasta > NOVOPlasty-fungal-mitogenome/K39/SRR5804130-fungal-mitogenome/temp.fa ; if [ -n "`ls NOVOPlasty-fungal-mitogenome/K39/SRR5804130-fungal-mitogenome/temp.fa`" ]; then python standardize_fas_names.py NOVOPlasty-fungal-mitogenome/K39/SRR5804130-fungal-mitogenome/temp.fa ; slim_fastg.py -F fungus_mt `ls NOVOPlasty-fungal-mitogenome/K39/SRR5804130-fungal-mitogenome/temp.fa | head -n1` --out-dir NOVOPlasty-fungal-mitogenome-eval --out-base SRR5804130 ; rm NOVOPlasty-fungal-mitogenome/K39/SRR5804130-fungal-mitogenome/temp.fa ; fi; 

# only keep the label files
rm NOVOPlasty-fungal-mitogenome-eval/*.fa
rm standardize_fas_names.py

# summary
python << END
import os
samples = [x for x in os.listdir("NOVOPlasty-fungal-mitogenome-eval") if x.endswith(".csv")]
with open("NOVOPlasty-fungal-mitogenome-eval/summary.tab", "w") as output_h:
    for s in samples:
        with open(os.path.join("NOVOPlasty-fungal-mitogenome-eval", s)) as input_h:
            head = input_h.readline()
            genes = []
            for line in input_h:
                if line.strip():
                    genes.extend(line.split("\t")[2].split(","))
            genes = sorted(set(genes))
            output_h.write(s.replace(".csv", "") + "\t" + str(len(genes)) + "\t" + ",".join(genes) + "\n")
END


